siriuspy/siriuspy/devices/injsys.py

```diff
@@ -303,10 +189,11 @@ def __init__(self):
         # ps devices
         self._psdevs = [PowerSupply(psn) for psn in self._psnames]
 
-        # trigger devices
-        self._trigdevs = [Trigger(trg) for trg in self._trignames]
+        # call base class constructor
+        super().__init__('', self._psdevs, hltiming=hltiming)
 
-        alldevs = tuple(self._psdevs + self._trigdevs)
+        # trigger devices
+        self._trigdevs = [self._hltiming.triggers(t) for t in self._trignames]
```

Comment:
bug: `[self._hltiming.triggers[t] for t in self._trignames]`

Reply:
done!